src/App.scss

```diff
@@ -21,3 +36,72 @@ form {
 		}
 	}
 }
+
+.slick-slider {
+	.slick-disabled {
+		display: none !important;
+	}
+	.slick-slide > div {
+		margin: 0 10px;
+	}
+	.slick-list {
+		margin: 0 -10px;
+	}
+}
+
+.list {
+	margin: 0 20px;
+	&__item {
```

Comment:
use '__' or '--' and get stick to that and also instead of pixels always try to use rem or em